### PR TITLE
All badges now show

### LINKED ISSUE #19: Add remaining chat badges
Not all chat badges are associated with an image. 

- [x] Associate all chat badges with an image

Story points: 1

### PR DESCRIPTION
**Pull request by FrankSalmick**

### CODE REVIEW

File: client/src/components/ChatTile.jsx [hunk 1]
```diff
@@ -66,3 +61,1913 @@ function ChatTile (props) {
 }
 
 export default ChatTile;
+
+// provided by twitch
```
Comment (HMcCullo) [inline client/src/components/ChatTile.jsx:65]: Is there a documentation link you could provide in comments?

File: client/src/components/ChatTile.jsx [hunk 2]
```diff
@@ -66,3 +61,1913 @@ function ChatTile (props) {
...
+const badges = {
```
Comment (HMcCullo) [inline client/src/components/ChatTile.jsx:67]: Is this something that could be moved to a JSON file so it doesn't clutter the JSX code? Genuine question, I'm not sure if JSON resource files are allowed in web dev.

**Review by HMcCullo — CHANGES_REQUESTED** [review]
Good code, a link to docs in the comments would be nice and potentially moving the large JSON blob to its own file. Let me know if those are not possible and I'll approve.

File: client/src/components/ChatTile.jsx (hunk 2 above)
Comment (FrankSalmick) [reply client/src/components/ChatTile.jsx:67]: not easily

File: client/src/components/ChatTile.jsx (hunk 1 above)
Comment (FrankSalmick) [reply client/src/components/ChatTile.jsx:65]: good idea, fixed

**Review by HMcCullo — APPROVED** [review]
Thanks for addressing my questions, looks good!

**Pull request merged**